File: Data Structures & Algorithms/two-integer-sum/submission-2.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> hmap = new HashMap<Integer,Integer>();
        int ans[] = new int[2];
        for(int i=0;i<nums.length;i++){
            int diff = target - nums[i];
            if(hmap.containsKey(nums[i])){
                System.out.println("inside if");
                ans[0] = hmap.get(nums[i]);
                ans[1] = i;
                break;
            }
            else{
                hmap.put(diff,i);
                System.out.println("inside else");
            }
        }
        return ans;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        ns = set(nums)
        ans = 0
        for i in ns:
            if i - 1 not in ns:
                cnt = 1
                while i + 1 in ns:
                    i += 1
                    cnt += 1
                ans = max(ans,cnt)
        return ans